src/Command/ImportSkillsCommand.php

```diff
@@ -34,37 +34,76 @@ protected function execute ( InputInterface $input, OutputInterface $output ) {
 		$skillsRepository = $manager->getRepository( Skill::class );
 
 		$slugs = [
+				'beekeeping',
+				'gardening',
+				'permaculture',
+				'nutrition',
+				'photography',
+				'video',
 				'botany',
-				'bryology',
 				'mycology',
+				'bryology',
 				'lichenology',
 				'entomology',
+				'hymenoptera',
+				'lepidoptera',
+				'diptera',
+				'beetles',
+				'saproxylo',
 				'mammalogy',
 				'ichthyology',
 				'ornithology',
 				'herpetology',
+				'batrachology',
 				'malacology',
 				'arachnology',
 				'carcinology',
-				'cetology',
+				'ketology',
+				'market gardening',
+				'viticulture',
+				'arboriculture',
+				'field crops',
+				'agroforestry',
+				'mixed farming',
+				'agricultural techniques plant cover',
+				'management techniques for grass and flower strips',
+				'hedge management techniques in agriculture',
+				'animal husbandry',
+				'technical landscape management',
+				'landscape design',
+				'hedge planting',
+				'soil life',
+				'renaturation',
+				'pond management',
+				'wetland management',
+				'rewilding',
 				'pedology',
 				'geology',
 				'oceanography',
 				'hydrology',
 				'hydromorphology',
 				'climatology',
 				'sociology',
-				'data-management',
+				'ethnology',
+				'ethology',
+				'data management',
 				'statistics',
+				'fundamental research',
+				'applied research',
 				'cartography',
-				'web-development',
+				'web development',
```

Comment:
Je pense qu'il faudrait garder la version avec `-` :

```suggestion
				'web-development',
```

et faire pareil pour tous les autres (enlever les espaces et caractères spéciaux et remplacer par `-`), pour éviter les soucis dans les fichiers `YML` (car ces chaînes y sont utilisées comme des clés).